cognite/neat/graph/transformers/_rdfpath.py

```diff
@@ -1,7 +1,49 @@
+from rdflib import RDF, Graph
+
+from cognite.neat.rules.analysis import InformationAnalysis
+from cognite.neat.rules.models._rdfpath import RDFPath, SingleProperty
+from cognite.neat.rules.models.information import InformationRules
+
 from ._base import BaseTransformer
 
 
 class ReduceHopTraversal(BaseTransformer):
     """ReduceHopTraversal is a transformer that reduces the number of hops to direct connection."""
 
     ...
+
+
+class AddAllReferences(BaseTransformer):
+    description: str = "Adds all references to specific property defined in Rules"
```

Comment:
I struggle with understanding this transformation from the description. I also think we should drop the description attribute and instead use the docstring attribute.